.travis.yml

```diff
@@ -11,7 +11,8 @@ before_script:
 
 script: >-
   ./config/travis/run-checks.sh &&
-  ./gradlew clean checkstyleMain checkstyleTest test coverage coveralls asciidoctor
+  ./gradlew clean checkstyleMain checkstyleTest headless test &&
+  ./gradlew test coverage coveralls asciidoctor
```

Comment:
I think this will cause the test suite to be run twice. Would it still work if these two lines are merged into something like `./gradlew clean checkstyleMain checkstyleTest headless test coverage coveralls asciidoctor` instead?

Reply:
oh woops i should remove 1 of the tests! Thanks for pointing out